src/main/java/com/thealgorithms/maths/KaprekarNumbers.java

```diff
@@ -0,0 +1,27 @@
+package com.thealgorithms.maths;
+
+public class KaprekarNumbers {
```

Comment:
We can add a function which will generate all the Kaprekar Numbers in a given range